zaza/model.py

```diff
@@ -1640,6 +1641,91 @@ async def _unit_status():
     async_block_until_wl_status_info_starts_with)
 
 
+async def async_block_until_unit_wl_status_info(
```

Comment:
Can we get a more specific name for the function?
async_block_until_unit_wl_status_regex?